code/game/objects/items/food/misc.dm

```diff
@@ -2,6 +2,14 @@
 ////////////////////////////////////////////OTHER////////////////////////////////////////////
 
 /obj/item/food/cheese
+	name = "cheese wheel"
+	desc = "A big wheel of delcious Cheddar."
+	icon_state = "cheesewheel"
+	food_reagents = list(/datum/reagent/consumable/nutriment = 10, /datum/reagent/consumable/nutriment/protein = 5, /datum/reagent/consumable/nutriment/vitamin = 5) //Hard cheeses contain about 25% protein
+	w_class = WEIGHT_CLASS_NORMAL
+	rat_heal = 35
```

Comment:
replace `rat_heal = 35` with `var/rat_heal = 35` because otherwise it is undeclaired